Library/Homebrew/brew.rb

```diff
@@ -159,8 +157,6 @@ def output_unsupported_error
   onoe e
   $stderr.puts e.backtrace if Homebrew.args.debug?
 
-  output_unsupported_error if Homebrew.args.HEAD?
```

Comment:
Still want this output.

Reply:
This is handled by making failed patches raise a `BuildError`.